stories/__tests__/SQForm.stories.test.js

```diff
@@ -151,7 +152,7 @@ describe('Tests for FormWithValidation', () => {
     //Ten Thousand Options
     userEvent.click(screen.getByLabelText(/ten thousand options/i));
     const firstOption = screen.getAllByRole('option')[0];
-    userEvent.click(screen.getAllByRole('option')[0]);
+    fireEvent.click(screen.getAllByRole('option')[0]);
```

Comment:
Question - I thought `userEvent` was preferred over `fireEvent`?

Reply:
It is, but it seems there was an update to `userEvent` that allows it to error if the target it's clicking isn't clickable because of things like `display: none` or `pointer-events: none`. Some of our tests ensure that nothing happens when an element is clicked and this error is preventing that. `fireEvent` is the low-level api that `userEvent` is built on, so we can still achieve the results we expect without being blocked by the fancy `userEvent` feature/error

Reply:
I feel like there should be an option to allow `userEvent` to click on an element that's somehow disabled without triggering the error, but I wasn't able to find one